sdk/tables/Azure.Data.Tables/MigrationGuide.md

```diff
@@ -0,0 +1,288 @@
+# Migration guide from Microsoft.Azure.Cosmos.Table to Azure.Data.Tables
+
+This guide is intended to assist in the migration to the Azure.Data.Tables client package from the legacy Microsoft.Azure.CosmosDB.Table package focusing on side-by-side comparisons for similar operations between the to versions.
+
+Familiarity with the Microsoft.Azure.CosmosDB.Table package is assumed. If you are new to the Azure Tables client library for .NET, please refer to the [README](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/README.md) and table client [samples](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/tables/Azure.Data.Tables/samples) rather than this guide.
```

Comment:
Nit: I would remove table client samples because _technically_ we have two clients.

Reply:
Do you mean change "table client samples" to "samples"?

Reply:
Yes